### PR TITLE
docs: add note about cli commands

### PR DESCRIPTION
**Pull request by FrederikSchlemmer**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #31073

## What is the new behavior?
Added a note:

"Learn more: This manual uses StackBlitz Generator for simplification. Take a look at Angular CLI to get an overview of our CLI commands."

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by FrederikSchlemmer** [comment]
@methodbox would like to have feedback from you!

File: aio/content/start/index.md [hunk 1]
```diff
@@ -204,6 +204,12 @@ You might have noticed that the product list component also defines a `products`
 
 We're going to create a new alert feature. The alert feature will take a product as an input. It will then check the product's price, and, if the price is greater than $700, it will display a "Notify Me" button that lets users sign up for notifications when the product goes on sale. 
 
+<div class="alert is-helpful">
+
+Learn more: This manual uses StackBlitz Generator for simplification. Take a look at [Angular CLI](cli) to get an overview of our CLI commands.
+
+</div>
+
```
Comment (jbogarthyde) [inline aio/content/start/index.md]: I think this needs more context, and should happen earlier. Perhaps, near the top:

We are using the StackBlitz Generator to show you a ready-made, simple application that you can examine and play with interactively. In actual development you will typically use the  [Angular CLI](guide/glossary#cli), a powerful command-line tool that lets you generate and modify applications. For more information, see the [CLI Overview](cli).

**Review by jbogarthyde — CHANGES_REQUESTED** [review]
I would make this more substantive,

**Comment by methodbox** [comment]
@FrederikSchlemmer

I agree with @jbogarthyde. The Angular team that commented on my original post may not agree, however.

It seems their view is that they want to obscure this from the user to avoid confusion.

I say let them know up front.

If the user is that confused by more clarity, it seems to me they may also be confused that StackBlitz is used and think it’s somehow required to use Angular. 

I doubt this is the case.

File: aio/content/start/index.md (hunk 1 above)
Comment (FrederikSchlemmer) [reply aio/content/start/index.md]: I've adopted the changes. There was a comment in the issue that it should only be a short note/link, so I only chosed this short note.
But I also prefer the detailed version at the beginning!

**Comment by FrederikSchlemmer** [comment]
@methodbox I've adopted the changes. As you said, I had implemented the suggestions of the Angular Team from the issue.
In my opinion, this approach now makes more sense.

File: aio/content/start/index.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ This tutorial introduces you to the essentials of Angular.
 It leverages what you already know about HTML and JavaScript&mdash;plus some useful Angular features&mdash;to build a simple online store application, with a catalog, shopping cart, and check-out form. 
 You don't need to install anything: you'll build the app using the [StackBlitz](https://stackblitz.com/ "StackBlitz web site") online development environment.
 
+<div class="alert is-helpful">
+
+We are using the StackBlitz Generator to show you a ready-made, simple application that you can examine and play with interactively. In actual development you will typically use the [Angular CLI](guide/glossary#command-line-interface-cli), a powerful command-line tool that lets you generate and modify applications. For more information, see the [CLI Overview](cli).
+
+</div>
```
Comment (jbogarthyde) [inline aio/content/start/index.md]: Thanks, I think this works.   
I would move it down below the "Create a new project" header, before the "click here".
Also, I think we need to not capitalize "Generator", that doesn't seem to be a specific StackBlitz term.

**Review by jbogarthyde — APPROVED** [review]
Some minor editorial changes, then LGTM

**Comment by methodbox** [comment]
Thanks, @mary-poppins!

I think the change is excellent, informative and doesn’t hinder the docs, yet it’s clearly placed in such a way to that it will stand out.

Good job, guys!

**Review by brandonroberts — APPROVED** [review]

**Pull request closed without merging**